source/index.txt

```diff
@@ -23,7 +23,7 @@ Introduction
 ------------
 The official MongoDB Node.js driver allows Node.js applications to
 connect to MongoDB and work with data. The driver features both
-callback-based and Promise-based interaction with MongoDB.
+callbacks and Promises when communicating with MongoDB.
```

Comment:
I would say "All asynchronous API in the driver supports returning Promises if a callback is not provided"

Did y'all decide that having a "fundamentals" page on this was not beneficial to our users? It seems like a `/fundamentals/promises` page would be really useful to answer a few common questions I get from users:
- what happens if I omit a callback?
- does the driver support async/await (answer: yes! anything that returns a Promise can be awaited)
- confusion around common async/await mistakes (forgetting `await` in `while (cursor.hasNext())`)

Reply:
That sounds like a good idea, I'll create a ticket for a new page on Promises. 

Our DevEd team had a discussion about whether to create a page on Promises, focused on what they were and the syntax related to them, but decided to link out to MDN docs instead. Given this information on common questions that you receive, I think there would be enough driver-relevant material to justify this new page.

Aside from the new page, I'm not sure I understand the recommendation you provided:
"All asynchronous API in the driver supports returning Promises if a callback is not provided"

Did you mean something like this:
"The official MongoDB Node.js driver allows Node.js applications to
connect to MongoDB and work with data. The driver features an asynchronous API which allows you to specify callbacks in method calls, or otherwise return values using Promises."

Reply:
New ticket for Fundamentals page on Promises https://jira.mongodb.org/browse/DOCSP-9706

Reply:
Yeah I think that's closer, I just wanted to be "Promise-positive" by saying that all the async api returns a Promise _if_ you don't provide a callback. Small difference, but we're trying to push our users to use async/await as much as possible

Reply:
Makes sense, I'll swap the mention order.